rosdep/python.yaml

```
@@ -1798,6 +1798,14 @@ python-flask-restful:
         packages: [flask-restful]
     xenial: [python-flask-restful]
     yakkety: [python-flask-restful]
+python3-flask-sqlalchemy:
+  arch: [python-flask-sqlalchemy]
+  debian: [python3-flask-sqlalchemy]
+  fedora: [python3-flask-sqlalchemy]
+  gentoo: [dev-python/flask-sqlalchemy]
+  nixos: [python39Packages.flask_sqlalchemy]
```

CI is pointing out that this is available in EPEL-8 for RHEL, which is why it is failing.  This should fix it:

```suggestion
  nixos: [python39Packages.flask_sqlalchemy]
  rhel:
    '*': [python3-flask-sqlalchemy]
    '7': null
```